lost/src/test/java/com/mapzen/android/lost/internal/MockEngineTest.java

```diff
@@ -124,7 +124,8 @@ public void setTrace_shouldNotRequireSpeed() throws Exception {
     assertThat(callback.locations.get(0).hasSpeed()).isFalse();
   }
 
-  @Test public void disable_shouldCancelTraceReplay() throws Exception {
+  @Test @Ignore("Failing")
```

Comment:
Seems related to all the other tests currently ignored with the comment `"Intermittently failing. Find a better way to test without Thread.sleep(100)"`. Let's use the same comment here and I'll add an issue to deal with all of these at some point.